emData/download.sh

```diff
@@ -319,13 +243,13 @@ do
 
     rm -rf ${target_dir}
     mkdir -p ${target_dir}/ReducedCombinedConfig
-    mkdir -p ${target_dir}/ReducedCombinedConfig2
+    ##mkdir -p ${target_dir}/ReducedCombinedConfig2
```

Comment:
@aryd I would prefer simply removing this line, since this configuration has been removed:
```suggestion
```

Reply:
Removed